From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: TreeKR <johanbag290@gmail.com>
Date: Sat, 30 Jan 2021 11:20:33 +0900
Subject: [PATCH] Debug IllegalStateException NotNull Fix


diff --git a/src/main/java/org/bukkit/Bukkit.java b/src/main/java/org/bukkit/Bukkit.java
index 68101a322ffab8ec28843386b79b8079576fa720..474e89f5f52f519a006511e645c6fddcc0ee3622 100644
--- a/src/main/java/org/bukkit/Bukkit.java
+++ b/src/main/java/org/bukkit/Bukkit.java
@@ -67,7 +67,7 @@ public final class Bukkit {
      *
      * @return Server instance being ran
      */
-    @NotNull
+
     public static Server getServer() {
         return server;
     }
