From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: norhu1130 <norhu1130@naver.com>
Date: Mon, 12 Apr 2021 21:16:12 +0900
Subject: [PATCH] WHAT BUG


diff --git a/src/main/java/net/minecraft/server/MinecraftServer.java b/src/main/java/net/minecraft/server/MinecraftServer.java
index 55811d164cb795758445b8057f615421391353ad..598b5904bbecfca2dd6139c150e89979bf079583 100644
--- a/src/main/java/net/minecraft/server/MinecraftServer.java
+++ b/src/main/java/net/minecraft/server/MinecraftServer.java
@@ -922,7 +922,7 @@ public abstract class MinecraftServer extends IAsyncTaskHandlerReentrant<TickTas
             time += t;
             total = total.add(x.multiply(dec(t)));
             if (++index == size) {
-                index = 0;i
+                index = 0;
 	    }
 
         }
